LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Webgenr Contributors
```

Comment:
Can "Webgenr Contributors" hold copyright even though we aren't a legal entity?

Reply:
any persons or corp can hold copyright, so yeah.
FYI -- I borrowed this concept from [tokio's license](https://github.com/tokio-rs/tokio/blob/master/LICENSE) 

which, as a side-note, may have originated from the need to not have individuals mentioned in the `toml` since this is (or at least used to be) a problem where crates, once published, cannot modify their meta-data
https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml